CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 - [#970](https://github.com/openshift/cluster-monitoring-operator/pull/970) Bump prometheus-operator v0.43.0.
 - [#971](https://github.com/openshift/cluster-monitoring-operator/pull/971) Enable `hwmon` in node-exporter for hardware sensor data collection
 - [#983](https://github.com/openshift/cluster-monitoring-operator/pull/983) Remove deprecated user workload configuration
+- [#994](https://github.com/openshift/cluster-monitoring-operator/pull/994) Collect PV volume plugin type and volume mode metrics
```

Comment:
Don't think this needs to be included in the release notes. We use the CHANGELOG to help our docs team to inform things users need to know. This is more for our internal devs so lets remove this.

Reply:
Removed